src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,240 @@
+- **Server-side batching:** Reduce costs and network overhead by grouping messages before delivery. For example, batching messages over a 500ms interval can reduce message costs by up to 8x in high-traffic streams.
+- **Rate limiting:** Control the flow of messages per user to maintain readability and prevent spam. Choose between per-user rate limits, global throttling, or adaptive algorithms.
+
+CODE HERE: Example of implementing per-user rate limiting logic
+
+PLACEHOLDER: Visual showing message spikes and how batching smooths delivery/costs
+
+**Decision point:**
+- What is the maximum message rate you want users to experience?
+- How will you balance real-time delivery with cost efficiency?
+
+## Authentication
+
+Authentication in livestream chat is a design decision. In some cases, you'll want to tie participants to their identity in your application. In others, anyone can join and participate. Ably Chat supports both of these scenarios - you don't need to go through the hassle of setting up users in Ably and trying to keep them in sync with your application. Every client has a clientID. If you want random users, just choose a random string. If you want to tie it to your application, then use the UUID of the user or some other well-known identifier.
+
+How you authenticate is also key. To balance security and experience, you want short-lived tokens that can be easily revoked if a users is misbehaving or needs their permissions changed, but also avoiding the overhead of changing permissions every 2 minutes. In production apps, we strongly recommend you don't use API keys for client-side authentication (server-side is OK) - these are long-lived and much more difficult to revoke in a hurry.
+
+With Ably Chat, authentication is best achieved using JSON Web Tokens (JWTs). These are tied to a particular clientID and come with a set of [capabilities](https://ably.com/docs/chat/setup#authentication) that control what a client can and cannot do. Ably's SDKs handle the timing and process of requesting a new token for your, refreshing it when it expires. All you need to do is provide a server-side endpoint that can generate the JWT for the client. This allows clients to use your existing authentication systems or user sessions to generate their Ably token.
+
+
+## Moderation: Protecting your community
+
+Effective moderation is essential for livestream chat to maintain a safe and engaging environment, prevent spam and inappropriate content, and ensure compliance with platform guidelines. While human moderators are valuable, they cannot scale to handle the high message volumes of popular livestreams - a single stream can generate thousands of messages per minute. Automated moderation solutions are therefore crucial for real-time content filtering at scale.
+
+Ably supports both inline (before-publish) and after-publish moderation workflows, making it easy to integrate with AI-powered moderation tools.
```

Comment:
Do we need to describe this as both "inline" and "before-publish"? 

"Ably supports moderating messages before they're published to the livestream, and retrospectively after they're published" for e.g.